app/views/dialogue.py

```diff
@@ -33,11 +33,11 @@
 
 @deprecated("Use BinaryChoiceDialog with exec() instead")
 def show_dialogue_confirmation(
-    title: Optional[str] = None,
-    text: Optional[str] = None,
-    information: Optional[str] = None,
-    details: Optional[str] = None,
-    button_text: Optional[str] = "Yes",
+    title: str | None = None,
+    text: str | None = None,
+    information: str | None = None,
+    details: str | None = None,
+    button_text: str | None = "Yes",
```

Comment:
Given that there is a default string for button_text, I don't think we need to make it | None, just make it str. Either that, make it default = None so that it defaults to the text from QMessageBox.StandardButton

Also, it doesn't seem like the param is used either way. However, realistically this method should be deprecated anyways since BinaryChoiceDialog should replace it and is much more powerful.

Reply:
@hk21702 I think that somebody sorta bastardized this along the way, because the function did work as intended when i made it. However, the new dialog is a better option, so I will implement that where it is needed and fully remove this function.